migrations/versions/abe953265132_.py

```diff
@@ -0,0 +1,49 @@
+"""empty message
+
+Revision ID: abe953265132
+Revises: af9215895b80
+Create Date: 2017-06-13 21:46:19.772583
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'abe953265132'
+down_revision = 'af9215895b80'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('speakers_calls',
+    sa.Column('id', sa.Integer(), nullable=False),
```

Comment:
I don't think this is a good way of migrating database. Doing this will cause loss in data I think. Can you rather try to rename the table instead of dropping and creating a new one?